scripts/tests/provisioning/test_rauc_provision.expect

```diff
@@ -49,6 +49,7 @@ set timeout 180
 # configure opkg
 expect {
 	-re "$RE_PROMPT" {
+		send "rm -f /etc/opkg/base-feeds.conf\r"
```

Comment:
Quick comment while I remember, how about removing dist.conf too?

Reply:
Darn; that completely slipped my mind. I'll see if I can remove that as well.